packages/suite/src/actions/suite/storageActions.ts

```diff
@@ -213,6 +214,8 @@ export const rememberDevice =
         if (!(await db.isAccessible())) return;
         if (!device || !device.features || !device.state) return;
         if (!remember) {
+            // eslint-disable-next-line @typescript-eslint/no-use-before-define
+            dispatch(forgetDeviceMetadataError(device));
```

Comment:
nitpick - why not just move it up or down? 🤷

Reply:
sometimes it is not possible. not sure how about this case. here I guess motivation might have been not making too many changes for review. I can try to move it as it is reviewed now.